src/alarms/notifications/App.js

```diff
@@ -97,7 +97,7 @@ class App extends Component {
     const alarmsTable = alarmRows.map((alarm, i) => {
       return (
         <AlarmRow
-          key={i}
+          key={alarm.uuid}
           alarm={alarm}
           loadAlarmsOnPage={this.loadAlarmsOnPage}
         />
```

Comment:
Het gebruiken van de index als key in een react component is bad practice.
Er moet een unieke identifier worden gebruikt.
In dit geval zorgde dit ervoor dat een component niet opnieuw gerendered werd, maar werd geupdate waardoor de state van dit component niet correct mee ging. Er werd een oud component geupdate dat toevallig dezelfde index had.
Een ctrl find van key={i in de raster management schermen leverde 31 resultaten op.
Deze ga ik waar mogelijk fixen.